pages/ecosystem.vue

```diff
@@ -40,9 +40,16 @@
               :target="`panel${tierName}`"
               :value="`${tierName}`"
             >
-              {{ tierName }}
+              {{ tierName }} ({{ getFilteredResultsCount(tierName) }})
             </bx-tab>
           </bx-tabs>
+          <div class="cds--row">
+            <div class="cds--col">
+              <div class="ecosystem__tiers__description">
+                {{ selectedTierDescription }}
+              </div>
+            </div>
+          </div>
```

Comment:
Do you need all these `div`s?

Reply:
Removed with https://github.com/Qiskit/qiskit.org/pull/3322/commits/c38bb12b5b3d96df6331c4d85bc7c8bc2f6b9abe